app/gateway/changelog.md

```diff
@@ -3,6 +3,493 @@ title: Kong Gateway Changelog
+setting. When using PKI for certificate verification in hybrid mode, you can now
+configure a list of Common Names allowed to connect to a control plane with the
+option. If not set, only data planes with the same parent domain as the control
+plane cert are allowed.
+
+#### Kong Manager
+
+* Fixed an issue where OIDC authentication into Kong Manager failed when used
+with Azure AD.
+
+* Fixed a performance issue with the Teams page in Kong Manager.
+
+* Fixed an issue with checkboxes in Kong Manager, where the checkbox for
+the OAuth2 plugin's `hash_secret` value was labelled as *Required* and users
+were not able to uncheck it.
+
+* Fixed an issue where Kong Manager was not updating plugin configuration
+ when attempting to clear the `service.id` from a plugin.
+
+* Fixes an issue with Route creation in Kong Manager, where a new route would
+default to `http` as the supported protocol. Now, creating a Route picks up
+the correct default value, which is `http,https`.
+
+* Kong Manager now accurately lists `udp` as a protocol option for Route and
+Service objects on their configuration pages.
+
+* Fixed an issue with Kong Manager OIDC authentication, which caused the error
+`“attempt to call method 'select_by_username_ignore_case' (a nil value)”`
+and prevented login with OIDC.
+
+* Fixed a latency issue with OAuth2 token creation. These tokens
+ are no longer tracked by the workspace entity counter, as the count is not
+ needed by the Kong Manager UI.
+
+* Fixed an issue where the plugin list table couldn't be sorted by the
+**Applied To** column.
+
+#### Dev Portal
+
+* When the SMTP configuration was broken or unresponsive, the API would respond
+with an error message that was a JavaScript Object (`[Object object]`) instead
+of a string. This happened when a user was registering on any given portal with
+broken SMTP. Now, if there is an error, the API responds with the string
+`Error sending email`.
+
+* The `/document_objects` and `/services/:id/document_objects` endpoints no
+longer accept multiple documents per service. This was an issue, as each service
+can only have one document. Instead, posting a document to one of these endpoints
+now overrides the previous document.
+
+#### Core
+
+* When the Router encounters an SNI FQDN with a trailing dot (`.`),
+  the dot will be ignored, since according to
+  [RFC-3546](https://datatracker.ietf.org/doc/html/rfc3546#section-3.1)
+  the dot is not part of the hostname.
+  [#8269](https://github.com/Kong/kong/pull/8269)
+
+* Fixed a bug in the Router that would not prioritize the routes with
+  both a wildcard and a port (`route.*:80`) over wildcard-only routes (`route.*`),
+  which have less specificity.
+  [#8233](https://github.com/Kong/kong/pull/8233)
+
+* The internal DNS client isn't confused by the single-dot (`.`) domain,
+  which can appear in `/etc/resolv.conf` in special cases like `search .`
```

Comment:
If possible, remove the space between search and the period: `search.`

Reply:
This is not a period. This is referring to the single-dot (`.`) domain, so `search .` means search everything in current/root location.